### DISCUSSION TITLE
French model :  tense of a verb is removed in version 3.x.

### DISCUSSION THREAD

**Discussion by paulo05100** [explosion/spaCy#9762] [Language Support]
Hello,
In French the tag_ for a verb no longer indicates the tense in version 3.x. The tag_ field is the same as pos_.

## Info about spaCy

(hello) paul@merry:~/plume/description$ python
Python 3.8.11 (default, Aug  3 2021, 15:09:35) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
>>> nlp=spacy.load('fr_core_news_lg')
>>> doc=nlp('le petit chat buvait du lait')
>>> for t in doc:
...  print( t.text,t.pos_,t.tag_)
... 
le DET DET
petit ADJ ADJ
chat NOUN NOUN
buvait VERB VERB
du ADP ADP
lait NOUN NOUN
>>> nlp=spacy.load('fr_core_news_sm')
>>> doc=nlp('le petit chat buvait du lait')
>>> for t in doc:
...  print( t.text,t.pos_,t.tag_)
... 
le DET DET
petit ADJ ADJ
chat NOUN NOUN
buvait VERB VERB
du ADP ADP
lait NOUN NOUN

## Info about spaCy

- **spaCy version:** 3.2.0
- **Platform:** Linux-4.15.0-123-generic-x86_64-with-glibc2.17
- **Python version:** 3.8.11
- **Pipelines:** fr_core_news_sm (3.2.0), fr_dep_news_trf (3.2.0), fr_core_news_lg (3.2.0)

**Comment by adrianeboyd** [comment]
Hi, the morphological features are no longer appended on the tags, but instead available as `token.morph` as a `MorphAnalysis` object, see: https://spacy.io/api/morphology#morphanalysis

**Reply by paulo05100** [reply]
Thank you very much for your responsiveness.
